include/swift/AST/ASTBridging.h

```diff
@@ -43,7 +43,7 @@ class BridgedASTContext;
 
 class BridgedIdentifier {
 public:
-  SWIFT_UNAVAILABLE("Use '.raw' instead")
+  SWIFT_NAME("Use '.raw' instead")
```

Comment:
The uses of `SWIFT_UNAVAILABLE` here are correct, #68353 is tracking adding `SWIFT_NAME` to functions that don't have optimal names for Swift. Most of the functions in this file have been appropriately annotated at this point, from a quick scan, I think the only remaining ones are the `Plugin_` functions that should be marked with `SWIFT_NAME` as methods of the plugin (and you would need to introduce a wrapper type in `ASTBridgingWrappers.def` for `LoadedExecutablePlugin` to achieve that).